android/assets/jsons/translations/English.properties

```diff
@@ -3636,116 +3636,6 @@ Furor Teutonicus =
 67% chance to earn 25 Gold and recruit a Barbarian unit from a conquered encampment = 
  # Requires translation!
 -[amount]% [param] unit maintenance costs = 
- # Requires translation!
-Berlin = 
- # Requires translation!
-Hamburg = 
- # Requires translation!
-Munich = 
- # Requires translation!
-Cologne = 
- # Requires translation!
-Frankfurt = 
- # Requires translation!
-Essen = 
- # Requires translation!
-Dortmund = 
- # Requires translation!
-Stuttgart = 
- # Requires translation!
-Dusseldorf = 
- # Requires translation!
-Bremen = 
- # Requires translation!
-Hannover = 
- # Requires translation!
-Duisburg = 
- # Requires translation!
-Leipzig = 
- # Requires translation!
-Dresden = 
- # Requires translation!
-Bonn = 
- # Requires translation!
-Bochum = 
- # Requires translation!
-Bielefeld = 
- # Requires translation!
-Karlsruhe = 
- # Requires translation!
-Gelsenkirchen = 
- # Requires translation!
-Wiesbaden = 
- # Requires translation!
-Munster = 
- # Requires translation!
-Rostok = 
- # Requires translation!
-Chemnitz = 
- # Requires translation!
-Braunschweig = 
- # Requires translation!
-Halle = 
- # Requires translation!
-Mצnchengladbach = 
```

Comment:
Still with strange Umlaut?

Reply:
Those are "-" lines, meaning: removed by this

Reply:
You're totally right 😅